src/main/java/com/libraryproject/librarysystem/dto/BookTypeResponseDto.java

```diff
@@ -0,0 +1,10 @@
+package com.libraryproject.librarysystem.dto;
+
+import java.util.Set;
+
+public record BookTypeResponseDto(
+        Long bookTypeId,
+        String name,
+        Set<BookResponseDto> books
```

Comment:
nie wiem czy to dto jest nam potrzebne. jesli już to zrezygnowałabym ze zwracania tego zbioru ksiązek. w prawdziwym życiu zawiesiłoby to prawdopodobnie system, należałoby tu uzyć paginacji (zbyt wiele ksiązek danego typu)

Reply:
Możliwe, że nie byłoby potrzebne, usunęłam